src/recordings.js

```diff
@@ -18,14 +20,20 @@ export async function fetchFirstRecordingURL(wordform) {
 
 
 /**
- * Render a list of speakers (in button form) for the user to interact with and hear the wordform pronounced in different ways.
+ * Render a list of speakers (in the form of a select) for the user to interact with and hear the wordform pronounced in different ways.
  */
 export function retrieveListOfSpeakers() {
   // get the value of the wordform from the page
   let wordform = document.getElementById('data:head').value
   let derivedURL = `${BASE_URL}/recording/_search/${wordform}`
 
-  let recordingsList = document.querySelector('.recordings-list')
+  let recordingsList = document.querySelector('.recordings-list') // to be deleted, but not yet because it'll break the refactoring 💀
+  
+  // select for our elements for playback and link-generation
+  let recordingsDropdown = document.getElementById('recordings-dropdown')
```

Comment:
So my philosophy with regards to IDs is that we should use them cautiously (or when HTML doesn't make give us any better option 💀 ).

I think the same thing can be accomplished by having just **ONE** ID for the entire `multiple-recordings` thing, then use child selectors to get the`<select>` and the `<button>`s and whatnot.